tests/test_github_stargazers.py

```diff
@@ -2,8 +2,8 @@
 import pytest
 import responses
 
-from github_stargazers.github_stargazers import GitHub
-from github_stargazers.github_stargazers import UsernameRepositoryError
+from github_stargazers.github import GitHub
```

Comment:
This class should be `test_github.py` since it only tests classes from the Github service. 

Also create test for `process`in this file.

Reply:
Yes, just missed that. I changed it to WIP.

Reply:
We decided to create tests for `process_command()` in a later PR which will refactor that part first.